src/sourmash/search.py

```diff
@@ -20,13 +20,19 @@ def calc_threshold_from_bp(threshold_bp, scaled, query_size):
     n_threshold_hashes = 0
 
     if threshold_bp:
+        if threshold_bp < 0:
+            raise TypeError("threshold_bp must be non-negative")
+
```

Comment:
oh, hah - this maybe addresses my question from your other PR :)

Reply:
yep!